File: 250424/폭탄 해제/bomb-removal.java
import java.util.Scanner;

class Bomb {
    String code;
    String color;
    int time;

    public Bomb(String code, String color, int time) {
        this.code = code;
        this.color = color;
        this.time = time;
    }
}

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String code = sc.next();
        String color = sc.next();
        int time = sc.nextInt();

        Bomb bomb = new Bomb(code, color, time);
        System.out.println("code : " + bomb.code);
        System.out.println("color : " + bomb.color);
        System.out.println("second : " + bomb.time);
        sc.close();
    }
}

File: 250424/폭탄 해제/bomb-removal.cpp
#include <iostream>
#include <string>
using namespace std;

class Bomb {
    public:
        string code;
        char color;
        int time;
        Bomb(string code, char color, int time) {
            this->code = code;
            this->color = color;
            this->time = time;
        }
};

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    string code;
    char color;
    int time;
    cin >> code >> color >> time;

    Bomb bomb = Bomb(code, color, time);

    cout << "code : " << bomb.code << "\n";
    cout << "color : " << bomb.color << "\n";
    cout << "second : " << bomb.time << "\n";
    return 0;
}